### PR TITLE
[doc] Add reference to DevelopmentTips.md in README.

### PR DESCRIPTION
**Pull request by varungandhi-apple**
Folks have pointed out that this needs to be more visible and I concur.

### CODE REVIEW

**Review by owenv — APPROVED** [review]

**Comment by CodaFi** [comment]
⛵️

**Comment by CodaFi** [comment]
@swift-ci please smoke test and merge

File: README.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ source code and up to 70 GB of disk space for the build artifacts with full
 debugging. Depending on your machine, a clean build can take a few minutes to
 several hours. Naturally, incremental builds are much faster.
 
+Once you are able to build things successfully and have a compile-test-debug
+loop going, check out the [development tips][docs/DevelopmentTips.md] for
```
Comment (benrimmington) [inline README.md]: This needs parentheses to create a link:

```diff
-loop going, check out the [development tips][docs/DevelopmentTips.md] for
+loop going, check out the [development tips](docs/DevelopmentTips.md) for
```

Comment (varungandhi-apple) [reply README.md]: Oops. Thanks, I have a second PR up for that.

**Pull request merged**